plugins/de.cognicrypt.core/META-INF/MANIFEST.MF

```diff
@@ -31,6 +32,7 @@ Bundle-ClassPath: .,
  lib/commons-io.jar,
  lib/commons-lang3.jar,
  lib/CryptoAnalysis.jar,
+ lib/de.darmstadt.tu.crossing.CryptSL.jar,
```

Comment:
Why have you added this dependency now? This should not be required as all CrySL related classes must be made available through CryptoAnalysis.